test/performance/benchmarks/broker-imc/continuous/100-broker-imc-setup.yaml

```diff
@@ -12,6 +12,80 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-mako
+  namespace: default
+data:
+  dev.config: |
```

Comment:
This should be `prod.config`, as in https://github.com/knative/eventing/blob/496ca0742672922a4b2b3b80e799926dc00c7a97/vendor/knative.dev/test-infra/scripts/performance-tests.sh#L61 we set the environment to `prod`.
A bit confusing though.

Reply:
I'll create a new PR to update these. The metrics are wrong anyway (the perf test image now reports `det` and `pet` instead of `ft`)